x-pack/plugins/security_solution/common/endpoint/data_loaders/index_alerts.ts

```diff
@@ -23,52 +24,55 @@ function delay(ms: number) {
  * @param numAlerts
  * @param options
  */
-export async function indexAlerts({
-  client,
-  eventIndex,
-  alertIndex,
-  generator,
-  numAlerts,
-  options = {},
-}: {
-  client: Client;
-  eventIndex: string;
-  alertIndex: string;
-  generator: EndpointDocGenerator;
-  numAlerts: number;
-  options: TreeOptions;
-}) {
-  const alertGenerator = generator.alertsGenerator(numAlerts, options);
-  let result = alertGenerator.next();
-  while (!result.done) {
-    let k = 0;
-    const resolverDocs: Event[] = [];
-    while (k < 1000 && !result.done) {
-      resolverDocs.push(result.value);
-      result = alertGenerator.next();
-      k++;
+export const indexAlerts = usageTracker.track(
```

Comment:
most of these show changes, but its mostly around wrapping them with `usageTracker.track()`